src/stories/Sidebar.stories.js

```diff
@@ -1,40 +1,132 @@
-/* eslint-disable no-multi-str */
+import { action } from '@storybook/addon-actions';
+import UnnnicSidebar from '../components/Sidebar/index.vue';
 
-import UnnnicSidebar from '../components/Sidebar/Sidebar.vue';
-import UnnnicSidebarItem from '../components/Sidebar/SidebarItem.vue';
-import UnnnicSidebarMenu from '../components/Sidebar/SidebarMenu.vue';
+const items = [
+  {
+    label: 'Item 1',
+    icon: 'tune',
+  },
+  {
+    label: 'Item 2 Grouped',
+    icon: 'forum',
+    children: [{ label: 'Child 1' }, { label: 'Child 2' }],
+  },
+  {
+    label: 'Item 3',
+    icon: 'tune',
+  },
+  {
+    label: 'Item 4 Grouped icons',
+    icon: 'tune',
+    children: [
+      { label: 'Child 1', icon: 'abc' },
+      { label: 'Child 2', icon: 'abc' },
+    ],
+  },
+];
+
+const setup = (args) => {
+  const handleNavigate = action('navigate');
+  const setActiveItem = (newActive) => {
+    const itemIndex = items.findIndex(
+      (el) => el.label === newActive.item.label,
+    );
+
+    const childIndex = newActive.child
+      ? items[itemIndex].children?.findIndex(
+          (child) => child.label === newActive.child.label,
+        )
+      : null;
+
+    const updatedActive = { itemIndex, childIndex };
+
+    args.active = updatedActive;
+
+    handleNavigate(newActive);
+
+    return updatedActive;
+  };
+
+  return { args, items, setActiveItem };
+};
 
 export default {
   title: 'Example/Sidebar',
   component: UnnnicSidebar,
+  decorators: [
+    () => ({
+      template: '<div ><story style="background: #fcfcfc !important;" /></div>',
+    }),
+  ],
```

Comment:
You can add the args here and remove of the rest of variations, avoiding redundant code.
```suggestion
  ],
  args: {
    items,
    width: '300px',
    active: { itemIndex: 1, childIndex: 0 },
  },
```